odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/DungeonWaypoints.kt

```diff
@@ -60,7 +85,68 @@ object DungeonWaypoints : Module(
     }
     private val debugWaypoint: Boolean by BooleanSetting("Debug Waypoint", false).withDependency { DevPlayers.isDev }
 
-    data class DungeonWaypoint(val x: Double, val y: Double, val z: Double, val color: Color, val filled: Boolean, val depth: Boolean, val size: Double)
+    data class DungeonWaypoint(
+        val x: Double,
+        val y: Double,
+        val z: Double,
+        val color: Color,
+        val filled: Boolean,
+        val depth: Boolean,
+        val size: Double,
+        val title: String?
+    )
+
+    class GuiSign : GuiScreen() {
```

Comment:
Make this an object instead of a class, you can still achieve the same result, look at TermSim objects if you need.